packages/docsify-changelog-plugin/src/index.js

```diff
@@ -33,11 +33,213 @@ const plugin = (hook, vm) => {
+    , document.documentElement.style.setProperty(`--changelog-header-h2-text`, `${u.header.h2_text}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h3-text`, `${u.header.h3_text}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h4-text`, `${u.header.h4_text}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h1-background`, `${u.header.h1_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h2-background`, `${u.header.h2_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h3-background`, `${u.header.h3_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h4-background`, `${u.header.h4_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h1-size`, `${u.header.h1_size}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h2-size`, `${u.header.h2_size}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h3-size`, `${u.header.h3_size}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h4-size`, `${u.header.h4_size}`)
+    , document.documentElement.style.setProperty(`--changelog-text-all`, `${u.text.all}`)
+    , document.documentElement.style.setProperty(`--changelog-text-strong`, `${u.text.strong}`)
+    , document.documentElement.style.setProperty(`--changelog-text-strong-background`, `${u.text.strong_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-p`, `${u.text.p}`)
+    , document.documentElement.style.setProperty(`--changelog-text-p-background`, `${u.text.p_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-blockquote`, `${u.text.blockquote}`)
+    , document.documentElement.style.setProperty(`--changelog-text-blockquote-background`, `${u.text.blockquote_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-list`, `${u.text.list}`)
+    , document.documentElement.style.setProperty(`--changelog-text-list-background`, `${u.text.list_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-list-icon`, `${u.text.list_icon}`)
+    , document.documentElement.style.setProperty(`--changelog-link-text`, `${u.link.a_text}`)
+    , document.documentElement.style.setProperty(`--changelog-link-text-hover`, `${u.link.text_hover}`)
+    , document.documentElement.style.setProperty(`--changelog-link-text-transition`, `${u.link.text_transition}`)
+    , document.documentElement.style.setProperty(`--changelog-link-background`, `${u.link.background}`)
+    , document.documentElement.style.setProperty(`--changelog-link-background-hover`, `${u.link.background_hover}`)
+    , document.documentElement.style.setProperty(`--changelog-link-background-transition`, `${u.link.background_transition}`);
+    e.config.changelog = u
+  })
+  
```

Comment:
can you please revert back to old style 

PR styling

![image](https://user-images.githubusercontent.com/26347874/76892415-b6492080-68b0-11ea-8771-1bde5d098803.png)

Old one 

![image](https://user-images.githubusercontent.com/26347874/76892453-c103b580-68b0-11ea-99d5-ed7295056259.png)


change it to old style

Reply:
revert the button hover style as well

PR style

![image](https://user-images.githubusercontent.com/26347874/76892566-e8f31900-68b0-11ea-92dd-2c4d54961729.png)

Old 

![image](https://user-images.githubusercontent.com/26347874/76892613-fad4bc00-68b0-11ea-85bc-a35510d5a0ba.png)

Reply:
So links in the heading should have a uniform size?
I put the color of the <p> back to orange
and the color of the background shade to black alpha .2

Reply:
I had the font set to bold in order to stand out better against the background, is that another thing that should be left to choice?

Reply:
remove those borders and keep like the old one 
slightly decrease the font size  of the headers
scrollbar width should be decreased as well
the second header (`Bug fixes` one), keep like the old one